_dashboards/workspace/workspace-acl.md

```diff
@@ -151,3 +151,47 @@ To remove several collaborators simultaneously, follow these steps:
 2. Select the **Delete x collaborators** button.
 3. Review the confirmation modal that appears.
 4. Select **Confirm** to remove all selected collaborators from the table.
+
+## Configuring workspace privacy
+
+When permission control is enabled, workspace administrators can set who has access to a workspace, with the following three access levels:
+
+* **Private to collaborators (Default):** Gives only workspace collaborators can access the workspace. 
+* **Anyone can view:** Grants **Read only** permissions to all users in the workspace, allowing them to view workspace assets. 
+* **Anyone can edit:** Grants **Read and write** permissions to all users, allowing view, create, and update workspace assets.
+
+Collaborators are granted higher permissions when their individual access level differs from that set in the workplace settings. For example, if workspace privacy is set to “Anyone can edit”, any collaborator with read-only access level will also be able to edit the workspace assets.
+
```

Comment:
Line 163: This is unclear as written. Is the first sentence accurate on its own? And is the second sentence a good example of the first (by default, wouldn't any collaborator have access to a workspace set to "Anyone can edit"?

Reply:
I think you're right, the second example confuses things. It already makes sense that someone with a higher access level would have more permissions.

Alt suggestion:

```suggestion
Users at the collaborator-level are granted additional permissions even when their access level differs from that of the workspace.
```

Reply:
Works for me, but removed a hyphen.